tests/chain_tests/plugins/tags/tags_tests.cpp

```diff
@@ -209,4 +213,131 @@ SCORUM_TEST_CASE(create_two_posts_with_same_tags)
 
 BOOST_AUTO_TEST_SUITE_END()
 
+BOOST_AUTO_TEST_SUITE(calc_pending_payout)
+
+template <asset_symbol_type SymbolType> struct stub_fund
+{
+    asset activity_reward_balance = asset(0, SymbolType);
+    fc::uint128_t recent_claims = 0;
+    curve_id author_reward_curve;
+
+    asset_symbol_type symbol() const
+    {
+        return activity_reward_balance.symbol();
+    }
+};
+
+using stub_fund_scr = stub_fund<SCORUM_SYMBOL>;
+using stub_fund_sp = stub_fund<SP_SYMBOL>;
+
+SCORUM_TEST_CASE(do_not_throw_and_return_zero_when_discussion_is_empty)
```

Comment:
But the fund is empty as well. 
Would be better to rename "do_not_throw_and_return_zero_when_DATA_is_empty"? Or check separately